_collections/_information/advancement.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default-md
+title: Advancement
+subtitle: It is recommended that Scouts attempt to complete one or multiple requirements towards advancement at each meeting. Troop 370 has registered Merit Badge Counselors and Scout Instructors to help Scouts advance. View the list below for advancement resources.
+nav_order: 2
+redirect_from: advancement.html
+column-count: 2
+---
+## Merit Badge Counselors
+
+A list of merit badge counselors and their contact information can be found in the password protected members section of our site.
+
+[Merit badge counselors page](/mb-counselors.html){:.mdc-button.md}
+
+## Merit Badge Clinics
+
+Sign up for Merit Badge classes at meritbadge.info.
+
+[meritbadge.info](http://meritbadge.info/){:.mdc-button.md}
+
+## TroopMaster Web
+
+TroopMaster Web allows each scout to track his advancement online.
+
+[TroopMaster Web](https://tmweb.troopmaster.com/mysite/troop370atl){:.mdc-button.md}
+
+## Advancement Requirements
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[heading-style] Headings should use setext](https://app.codacy.com/app/Troop_370/troop370/pullRequest?prid=4642321)